src/main/java/Controller/Engine/Stockfish.java

```diff
@@ -0,0 +1,78 @@
+package Controller.Engine;
+
+import Model.Game;
+import Model.Pieces.PieceType;
+import Model.Position;
+
+import java.io.*;
+import java.util.Objects;
+
+import static java.util.Objects.isNull;
+
+public class Stockfish implements Runnable{
+
+    private Process process;
+    private String command;
+    private Game game;
+
+    public Stockfish(String fen, Game game){
+        this.game = game;
+        ProcessBuilder builder = new ProcessBuilder();
+        builder.command("stockfish_14.1_win_x64_avx2.exe");
```

Comment:
i would change this to `builder.command('stockfish')` and then in the readme explain that `stockfish` needs to be in the users path, along with download instructions.